profile/README.md

```diff
@@ -0,0 +1,29 @@
+[![logo](https://raw.githubusercontent.com/wmde/.github/main/images/WMDEGitHubBanner.png)](https://github.com/wmde)
+
+Wikimedia Deutschland is the German chapter of the global [Wikimedia movement](https://www.wikimedia.org/). Our technical focus is on [Wikidata](https://www.wikidata.org/), the [Wikibase ecosystem](https://wikiba.se/) and software that serves the German speaking and wider Wikimedia community.
+
+## Our projects
+
+### Wikidata
+
+A collaboratively edited multilingual knowledge graph that powers Wikipedia, Wikivoyage, Wiktionary, Wikisource, and many other sites and services beyond just Wikimedia projects! See [wikidata.org](https://www.wikidata.org).
+
+### Wikibase Suite
+
+Dockerized deployment of Wikibase, the technology behind Wikidata. It alows you to host your very own linked open data knowledge base. See [installation guide on MediaWiki](https://www.mediawiki.org/wiki/Wikibase/Suite).
```

Comment:
typo: allows

Reply:
Line 3: change to "German-speaking and the wider" (add hyphen and the word "the")
Line 9: remove "just" and change exclamation point to period (to avoid the scent of ad-speak)
Line 13: change to "A Dockerized..."; change to "allows" with two Ls; remove "very"; change to "our installation guide"

Reply:
4715b50 implemented the changes above :)